modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/codegen/CodeGenerator.java

```diff
@@ -493,9 +493,12 @@ public void visit(BLangReturn returnNode) {
             expr = returnNode.exprs.get(i);
             this.genNode(expr, this.env);
             if (expr.isMultiReturnExpr()) {
-                BLangInvocation invExpr = (BLangInvocation) expr;
-                for (int j = 0; j < invExpr.regIndexes.length; j++) {
-                    emit(this.typeTagToInstr(invExpr.types.get(j).tag), i, invExpr.regIndexes[j]);
+//                BLangInvocation invExpr = (BLangInvocation) expr;
+//                for (int j = 0; j < invExpr.regIndexes.length; j++) {
+//                    emit(this.typeTagToInstr(invExpr.types.get(j).tag), i, invExpr.regIndexes[j]);
+                MultiReturnExpr invExpr = (MultiReturnExpr) expr;
```

Comment:
Shall we remove these commented code lines?

Reply:
Ack,